source/base/Profiler.ooc

```diff
@@ -65,6 +65,7 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
+		This _profilers[0] printResults()
```

Comment:
This will only print the result of one, not all (if you have had several).
```ooc
This _profilers apply(func (profiler: This) { profiler printResults() })
```